dts/arm/microchip/mec1501hsz.dtsi

```diff
@@ -344,71 +344,71 @@
 			pcrs = <1 4>;
 			label = "PWM_0";
 			status = "disabled";
-			#pwm-cells = <1>;
+			#pwm-cells = <2>;
```

Comment:
Same updates needed in mec172xnsz.dtsi:
https://github.com/zephyrproject-rtos/zephyr/blob/07d433878e780ba4ab05399aa13d0e32538c7359/dts/arm/microchip/mec172xnsz.dtsi#L595

Reply:
done